samples/app-anonymous-users/nodejs/ClientApp/src/components/share-to-meeting.jsx

```diff
@@ -0,0 +1,26 @@
+// <copyright file="share-to-meeting.jsx" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// </copyright>
+
+import React, { useEffect } from "react";
+
+// Default browser page from where content can directly shared to meeting.
+const ShareToMeeting = props => {
+
+    useEffect(() => {
+        const script = document.createElement("script");
+        script.src = "https://teams.microsoft.com/share/launcher.js";
+        document.body.appendChild(script);
+    }, []);
+
+    return (
+        <>
+		    <div class="surface">
+                <h3 id="tabheader">Welcome to the app that support anonymous user support in teams meeting apps.</h3>
```

Comment:
Add auth related also

Reply:
We used the C# sample as well as this content. 